INSTALL.txt

```diff
@@ -7,7 +7,8 @@ Prerequisites:
                     gnupg flex bison gperf build-essential zip curl zlib1g-dev \
                     gcc-multilib g++-multilib libc6-dev-i386 lib32ncurses5-dev \
                     x11proto-core-dev libx11-dev lib32z-dev ccache libgl1-mesa-dev \
-                    libxml2-utils xsltproc unzip python-clang-5.0 gcc-5 g++-5 bc -y
+                    libxml2-utils xsltproc unzip python-clang-5.0 gcc-5 g++-5 bc \
+                    pyelftools pycryptodomex gcc-aarch64-linux-gnu -y
```

Comment:
It's better to replace `pyelftools pycryptodomex` by `python-pyelftools python3-pycryptodome`.
I found this after experiments with environment.

Reply:
> python-pyelftools python3-pycryptodome

done